efel/tests/test_basic.py

```diff
@@ -3176,6 +3176,75 @@ def test_AHP_depth_slow():
     )
 
 
+def test_AHP_depth_from_AP_begin_voltage():
+    """basic: Test AHP depth from AP begin voltage"""
+
+    import efel
+    efel.reset()
+
+    trace, time, voltage, stim_start, stim_end = load_data(
+        'mean_frequency1', interp=True)
+
+    features = [
+        "AHP_depth_from_AP_begin_voltage", "AHP_depth_abs", "AP_begin_voltage"
+    ]
+
+    feature_values = \
+        efel.getFeatureValues(
+            [trace],
+            features, raise_warnings=False)
+
+    AHP_depth_from_AP_begin_voltage = feature_values[0][
+        "AHP_depth_from_AP_begin_voltage"
+    ]
+    AHP_depth_abs = feature_values[0]["AHP_depth_abs"]
+    AP_begin_voltage = feature_values[0]["AP_begin_voltage"]
+
+    # we do this to mimic the cpp implementation, sometimes different AP are
+    # picked up  but as we consider averages, it's ok. To fix at some point.
+    expected = [v - ahp for v, ahp in zip(AP_begin_voltage, AHP_depth_abs)]
```

Comment:
Using numpy, we could just do `expected = AP_begin_voltage - AHP_depth_abs` I believe.

Reply:
no, this does not work, I explained in the comment above why, but I'm really unsure on how to fix it.

Reply:
I don't get your comment. What do you mean by 'sometimes, differnet APs are picked up'. That sounds pretty bad